web/satellite/vue.config.js

```diff
@@ -1,15 +1,21 @@
 // Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
+const path = require('path');
+
 module.exports = {
+    publicPath: "/static/dist",
```

Comment:
Not sure if thats ok to be an absolute path?

Reply:
That's route to our file server